tests/aws/services/apigateway/test_apigateway_basic.py

```diff
@@ -1756,13 +1669,34 @@ def test_apigw_call_api_with_aws_endpoint_url(aws_client, region_name):
 
 
 @pytest.mark.parametrize("method", ["GET", "ANY"])
-@pytest.mark.parametrize("url_function", [path_based_url, host_based_url])
-@markers.aws.unknown
+@pytest.mark.parametrize("url_type", [path_based_url, UrlType.HOST_BASED])
+@markers.aws.validated
+# TODO clean up the client instances. We might not need 4.
+#  We might also be ok not parametrizing the method, as it doesn't seem to be what we are testing here
 def test_rest_api_multi_region(
```

Comment:
it seems we have snapshots created for this test, but I can't see the `snapshot` fixture. Should we delete them from the snapshot file?